docs/other-topics/transactions.md

```diff
@@ -2,252 +2,241 @@
-
-  const user = await User.create({
-    firstName: 'Bart',
-    lastName: 'Simpson'
-  }, { transaction: t });
-
-  await user.addSibling({
-    firstName: 'Lisa',
-    lastName: 'Simpson'
-  }, { transaction: t });
-
-  // If the execution reaches this line, no errors were thrown.
-  // We commit the transaction.
-  await t.commit();
-
-} catch (error) {
-
-  // If the execution reaches this line, an error was thrown.
-  // We rollback the transaction.
-  await t.rollback();
-
-}
-```
-
-As shown above, the *unmanaged transaction* approach requires that you commit and rollback the transaction manually, when necessary.
-
-## Managed transactions
-
-Managed transactions handle committing or rolling back the transaction automatically. You start a managed transaction by passing a callback to `sequelize.transaction`. This callback can be `async` (and usually is).
+Managed transactions are started using [`sequelize.transaction`](pathname:///api/v7/classes/Sequelize.html#transaction).
+a method which accepts an async callback. This method works like this:
 
-The following will happen in this case:
-
-* Sequelize will automatically start a transaction and obtain a transaction object `t`
-* Then, Sequelize will execute the callback you provided, passing `t` into it
+* Sequelize will start a transaction
+* Then, Sequelize will execute the callback you provided
 * If your callback throws, Sequelize will automatically rollback the transaction
-* If your callback succeeds, Sequelize will automatically commit the transaction
-* Only then the `sequelize.transaction` call will settle:
-  * Either resolving with the resolution of your callback
-  * Or, if your callback throws, rejecting with the thrown error
-
-Example code:
+* If your callback finishes successfully, Sequelize will automatically commit the transaction
 
-```js
+```ts
 try {
-
-  const result = await sequelize.transaction(async (t) => {
-
+  const result = await sequelize.transaction(async () => {
+    // both of these queries will run in the transaction
     const user = await User.create({
       firstName: 'Abraham',
       lastName: 'Lincoln'
-    }, { transaction: t });
+    });
 
     await user.setShooter({
       firstName: 'John',
       lastName: 'Boothe'
-    }, { transaction: t });
+    });
 
     return user;
-
   });
 
   // If the execution reaches this line, the transaction has been committed successfully
   // `result` is whatever was returned from the transaction callback (the `user`, in this case)
 
-} catch (error) {
-
+} catch {
   // If the execution reaches this line, an error occurred.
   // The transaction has already been rolled back automatically by Sequelize!
-
 }
 ```
 
-Note that `t.commit()` and `t.rollback()` were not called directly (which is correct).
+### Using save points
 
-### Throw errors to rollback
+In dialects that support save point, you can create a new save point by nesting managed transaction calls:
```

Comment:
How do people know which dialects support save points?

Reply:
We can add a support table like here: https://640374baea7cbb5439a3990a--sequelize-site.netlify.app/docs/v7/querying/operators/#array-operators

Reply:
That would be nice

Reply:
Oh yeah, I forgot about that! Can we do that in the other PR as that support able component is not available in this PR?

Reply:
That's fine with me